src/main/scala/cromwell/binding/command/ParameterCommandPart.scala

```diff
@@ -90,6 +90,7 @@ case class ParameterCommandPart(wdlType: WdlType, name: String,
      * turned into strings, but a more sophisticated solution will be
      * needed for compound types */
     paramValueEvaluated match {
+      case str:WdlString if str.value.isEmpty => ""
```

Comment:
Curiousity Q -> is the `isEmpty` also guarding against `null` (as in is there an expectation that it could actually be null) and if so can we root it out at the java/scala border?

Along those lines if null doesn't enter into it, something like `case WdlString("") => ""` should work and manage to be both more clear & terse at the same time. But if `null` is a concern that's clearly not so good :)